### PR TITLE
Added new input variable 'name'

### PR DESCRIPTION
**Pull request by juan-acevedo-ntt**
Fixes #000 

Changes proposed in the pull request:

Adds a new variable "name" that overrides "prefix". When this new variable is set the name of the load balancer will be set to match the variable value. It enables custom resource naming including the recommended Azure naming convention (https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations). The name of the private ip will also change to "pip-${name}"

Also includes https://github.com/Azure/terraform-azurerm-loadbalancer/pull/34

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -3,9 +3,14 @@ data "azurerm_resource_group" "azlb" {
   name = var.resource_group_name
 }
 
+locals {
+  lb_name  = var.name != "" ? var.name : format("%s-lb", var.prefix)
+  pip_name = var.name != "" ? format("pip-%s", var.name) : format("%s-publicIP", var.prefix)
```
Comment (yupwei68) [inline main.tf:8]: Instead of setting the name to "pip-${var.name}", would you create another variable named "pip_name" for this?

**Review by yupwei68 — COMMENTED** [review]
Thanks for your contribution! 🚀

File: main.tf (hunk 1 above)
Comment (juan-acevedo-ntt) [reply main.tf]: Done

**Review by yupwei68 — APPROVED** [review]
LGTM!

**Pull request merged**